tests/forbiddenword.wrong

```diff
@@ -2,3 +2,6 @@ bars
 foos
 foobar
 barfoo
+Kg
+KG
+Cm
```

Comment:
CM is currently also considered misspelled, but I'm not sure if it's correct. After all, its lowercase counterpart `cm` isn't forbidden.